opennms-pris-plugins/opennms-pris-plugins-jdbc/src/test/java/org/opennms/opennms/pris/plugins/jdbc/source/JdbcSourceTest.java

```diff
@@ -67,13 +66,13 @@ public void setUp() throws ClassNotFoundException, SQLException {
         Statement stmnt = connection.createStatement();
         stmnt.executeUpdate(SQL_CREATE_ALL);
 
-        insertRow("1", "node1", "", "", "", "192.168.0.1", "P", "1", "description1", "city1", "state1", "service1", "category1");
-        insertRow("2", "node2", "ParentNodeLabel", "", "",  "192.168.0.2", "P", "3", "description2", "city2", "state2", "service2", "category2");
-        insertRow("3", "node3", "", "ParentId", "ParentSouce", "192.168.0.3", "P", "1", "description3", "city3", "state3", "service3", "category3");
+        insertRow("1", "node1", "", "", "", "", "192.168.0.1", "P", "1", "description1", "city1", "state1", "service1", "category1");
```

Comment:
Shouldn't this be `null` instead of empty string. The check in `JdbcSource.java` checks only for `null`.